demos/check_for_unreads.js

```diff
@@ -8,10 +8,13 @@ async function main() {
     await bot.init({username: process.env.KB_USERNAME, paperkey: process.env.KB_PAPERKEY, verbose: false})
     const res = await bot.chatList({
       showErrors: true,
-      unreadOnly: true,
     })
-    const unreadCount = res.conversations.filter(c => c.unread).length
-    console.log(`You have ${unreadCount} unread conversations out of ${res.conversations.length} total`)
+    if (res) {
```

Comment:
@nathunsmitty good catch on accounts without conversations.

Here's a thought. Why not maintain our return type that `chat.list` will always return an array and default to an empty array in `chatList` (what will soon be `chat.list`).

So if the return value from `keybase chat list` is `null` or `undefined`, just return empty list `[]`.

This was having an inbox with no unreads can be handled the same as the case where there are no conversations in the inbox at all.